english/Egypt-the-4000-years-of-civilization/25th-january-revolution/revolution.md

```diff
@@ -0,0 +1,4 @@
+	Teach your childern that mohamed hosny moubarak "The Ousted president" was just a pain killer.
+The Egyptian revolution of 2011, locally known as the January 25 Revolution and as the Egyptian Revolution of Dignity,
+began on 25 January 2011 and took place across all of Egypt. The date was set by various youth groups to coincide with
+the annual Egyptian "police day" as a statement against increasing police brutality during the last few years of Mubarak's presidency.	
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.